app/Http/Controllers/UserAdminController.php

```diff
@@ -251,12 +253,16 @@ public function showForm(Request $request, $id)
 
         $user = User::findOrFail($id);
 
+        if ($user->membershipContract) {
+            Storage::download($user->membershipContract->generatePath(), $user->name . ' membership contract.pdf');
```

Comment:
Either also have this support the "printer-host" functionality, or remove the "printer-host" functionality from this URL altogether please.